packages/manager/modules/pci/src/projects/project/storages/databases/database/pools/information/information.html

```diff
@@ -0,0 +1,86 @@
+<oui-modal
+    data-heading="{{:: 'pci_databases_pools_information_title' | translate }}"
+    data-on-dismiss="$ctrl.goBack()"
+>
+    <div class="mb-3 information">
+        <dl>
+            <div class="row my-2 py-2">
```

Comment:
**question:** Relates to my previous comment regarding custom style. Not sure to understand why you are using Grid and Definition List. Existing Definition List couldn't suit your needs?

Reply:
**question:** Why are you using a grid? Please prefer stick with guidelines around definition list.
see: https://ovh.github.io/ovh-ui-kit/?path=/story/design-system-content-descriptions--default